packages/controls-navigator/src/NavigatorControl.vue

```diff
@@ -0,0 +1,293 @@
+    }
+  }
+};
+const isTraveled = (index) => {
+  travel.value[currentStep.value] = true;
+  return index !== currentStep.value && !!travel.value[index];
+};
+const getNavigateClass = () => {
+  return navigationType.value === "tab" ? "PTNavigationTabClass" : "PTNavigationStepperClass";
+};
+const getComplete = (index) => {
+  // numOfSteps is number of steps to go back
+  if (navigationType.value === "tab") {
+    return isTraveled(index);
+  }
+  return currentStep.value > index;
+};
+
+defineExpose({
+  gotoStep,
+  steps,
+  currentStep,
+  getStepClass,
+});
+</script>
+
+<style lang="scss">
+div.PTNavigationTabClass {
+  div.v-stepper .v-stepper-item {
+    padding-left: 0px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    opacity: 1 !important;
+    /* Step bullet  */
+    .v-stepper-item__avatar {
+      font-size: 0px;
+
+      background-color: var(
+        --vscode-editor-background,
+        white
+      ) !important; // Override Vuetifys theme specific selector `.theme--dark.v-stepper .v-stepper__step:not(.v-stepper__step--active):not(.v-stepper__step--complete):not(.v-stepper__step--error) .v-stepper__step__step`
+
+      //background-color: transparent !important; // Required since  Vuetify `.v-application .primary` adds background-color !important
+      color: var(--vscode-foreground, #616161);
+      border: 1px solid;
+      height: 10px !important;
+      width: 10px !important;
+      min-width: 10px;
+      margin-right: 10px;
+      transition: 0.3s ease-in-out;
+      .v-icon.v-icon {
+        font-size: 0;
+      }
+    }
+    /* Step label */
+    .v-stepper-item__title {
+      font-weight: bold;
+      font-size: 14px;
+      line-height: 17px;
+      color: var(
+        --vscode-foreground,
+        #616161
+      ) !important; // Required since Vuetify adds inline color for selector: `.theme--dark.v-stepper .v-stepper__label`
+      transition: 0.3s ease-in-out;
+    }
+    &--complete {
+      .v-stepper-item__avatar {
+        background-color: var(--vscode-foreground, #616161) !important;
+        border-color: var(
+          --vscode-foreground,
+          #616161
+        ) !important; // Required since Vuetify adds border-color !important for selector `.v-application .primary`
+      }
+    }
+    &--selected {
+      .v-stepper-item__title {
+        color: var(
+          --vscode-pickerGroup-foreground,
+          #0066bf
+        ) !important; // Required since Vuetify adds inline color for selector: `.theme--dark.v-stepper .v-stepper__label`
+        transition: 0.3s ease-in-out;
+      }
+      .v-stepper-item__avatar {
+        background-color: var(--vscode-pickerGroup-foreground, #0066bf) !important;
+        border-color: var(
+          --vscode-pickerGroup-foreground,
+          #0066bf
+        ) !important; // Required since Vuetify adds border-color !important for selector `.v-application .primary`
+      }
+    }
+  }
+}
+
+div.PTNavigationStepperClass {
```

Comment:
same here for removing `PT` from the name

Reply:
has removed it.